src/cpp/feelpp/README.adoc

```diff
@@ -0,0 +1,35 @@
+:feelpp: Feel++
+= Heat examples
+:toc: macro
+
+toc:[]
+
+== feelpp heat with json
+
+
+
+[source,shell]
+----
+# configure
+cmake --preset default
+
+# build
+cmake --build --preset default heat.j -j4
+
+# go in build directory 
+cd build/default/src/cpp/feelpp
+
+# copy case files to current build directory
+make feelpp_add_testcase_
+----
+
+=== 2D
+
+.2D one  thermal fin 
+[source,shell]
+----
+# in sequential
+../../../bin/heat.j --config-file cases/rht/fin1/fin.cfg 
+# in parallel
+mpirun -np 2 ../../../bin/heat.j --config-file cases/rht/fin1/fin.cfg
+----
```

Comment:
The command for the execution of the parareal version of the problem should be added. It should be 

```
=== 2D with parareal
The code is run on 6 processors, 2 for the coarse integrator and 2x2 for the fine integrators. The mesh should be partitioned in advance, but for this case the partitioning is already provided in the repository. 

.2D thermal fin parareal on 6 processors
[source, shell]
----
mpirun -np 6 ../../../bin/heat.e --config-file cases/rht/fin1/fin.cfg
----
```